src/sb_pointcloud_processing/include/DBSCAN.h

```diff
@@ -16,6 +16,10 @@ using namespace std;
 using namespace std::tr1;
 
 class DBSCAN {
+  public:
+    enum Plane { XY, YZ };
```

Comment:
Does `DBSCAN` only work in 2D? Quick doc comment for this enum would be helpful.